lib/nexpose/version.rb

```diff
@@ -1,4 +1,4 @@
 module Nexpose
   # The latest version of the Nexpose gem
-  VERSION = '3.1.0'
+  VERSION = '3.1.1'
```

Comment:
Yeah, let's not actually include the version file in a pull request. We'll bump the version when we release the next version of the gem.